### PR TITLE
data: phatwillis.json

### PR DESCRIPTION
**Pull request by phatwillis**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/phatwillis.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "group": "Github",
+      "name": "Github: Check out my Commits!",
+        "url": "http://Github.com/phatwillis",
+      "icon": "FaGithub"
+    },
+     {
+      "group": "  Twitter",
+      "name": "Twitter: Follow Me on Twitter!",
+        "url": "http://Twitter.com/ogaga_williams",
+      "icon": "FaTwitter"
```
Comment (sital002) [inline data/phatwillis.json:15]:
```suggestion
      "icon": "FaTwitter",
```
There is a comma missing here

File: data/phatwillis.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+       "color": "#000000"
+    },
+     {
+      "group": "Linkedin",
+      "name": "Linkedin: Let's Connect!",
+        "url": "https://www.linkedin.com/in/williams-michael-63686023a",
+      "icon": "FaLinkedin"
```
Comment (sital002) [inline data/phatwillis.json:22]:
```suggestion
      "icon": "FaLinkedin",
```
Please add a comma here

File: data/phatwillis.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+       "color": "#000000"
+    },
+    {
+      "group": "Behance",
+      "name": "Behance: Check out my Designs!",
+        "url": "https://www.behance.net/williamsmicheal",
+      "icon": "FaBehance"
```
Comment (sital002) [inline data/phatwillis.json:29]:
```suggestion
      "icon": "FaBehance",
```

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

**Review by SaraJaoude — APPROVED** [review]
I have committed the changes to make the Json pass and I will now merge this PR.

**Pull request merged**